packages/Webkul/Shop/src/Resources/views/customers/account/orders/pdf.blade.php

```diff
@@ -192,7 +192,7 @@
 
                                 <div class="row">
                                     <span
-                                        class="label">@lang('shop::app.customer.account.order.view.invoice-date'):
+                                        class="label">@lang('admin::app.sales.invoices.invoice-pdf.date'):
```

Comment:
Same here

Reply:
@amit-webkul,
We have used the 'admin' lang in most places. Do we need to update it everywhere, or is it necessary only for those two specific locations you mentioned?

Reply:
For separation of concern, in the shop package, there will be no admin-related lang should be present. Same vise versa for admin also.

Reply:
If specific language keys are not present then add your key to the shop package also.